python/ray/data/datasource/datasource.py

```diff
@@ -244,8 +245,10 @@ def make_block(start: int, count: int) -> Block:
             if block_format == "arrow":
                 import pyarrow as pa
 
+                # Use the reserved column name for representing tensor arrays in
+                # tabular format.
                 return pa.Table.from_arrays(
-                    [np.arange(start, start + count)], names=["value"]
+                    [np.arange(start, start + count)], names=[VALUE_COL_NAME]
```

Comment:
This isn't a tensor array FYI, this is what you'd get with e.g. `ray.data.range_table()`, and we ensure that the column name is human readable (like `"value"`), not our internal `TENSOR_COL_NAME`. Could you revert this back?